aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditHorizontalExample.js

```diff
@@ -0,0 +1,146 @@
+import React, { Fragment, useContext, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import {
+  Anchor,
+  Box,
+  Button,
+  Form,
+  Heading,
+  MaskedInput,
+  NameValueList,
+  NameValuePair,
+  TextInput,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import {
+  FormGrid,
+  NameValueListFormField,
+  NameValueListFormLabel,
+} from './components';
+import { companyInfomation } from '../data';
+
+export const NameValueListEditHorizontalExample = () => {
+  const theme = useContext(ThemeContext);
+  const size = useContext(ResponsiveContext);
+  const [currentData, setCurrentData] = useState(companyInfomation);
+  const [tempData, setTempData] = useState(currentData);
+  const [edit, setEdit] = useState(false);
+
+  const Wrapper = size !== 'small' ? Fragment : Box;
+
+  return (
+    <Box gap="medium">
+      <Box justify="between" width="large" direction="row" align="start">
+        <Box gap="xsmall">
+          <Heading level={2} size="small" margin="none">
+            Company Information
+          </Heading>
+          <Text size="large">Add your company information.</Text>
+        </Box>
+        {!edit && (
+          <Button secondary label="Edit" onClick={() => setEdit(true)} />
+        )}
+      </Box>
+      {!edit ? (
+        <NameValueList>
+          {Object.entries(currentData).map(([name, value]) => (
+            <NameValuePair key={name} name={<Text weight="bold">{name}</Text>}>
+              {name === 'Email' ? (
+                <Anchor label={value.value} href={`mailto:${value.value}`} />
+              ) : (
+                <Text {...theme.global.input.font}>{value.value || '--'}</Text>
+              )}
```

Comment:
Candidate for custom render function pattern.

Also, do not see Email in our data.

Reply:
Thanks, this is lingering from a different example.

Reply:
Updated to support this custom render.